#Message that sends error of properties
uint8 property_id
bool blue
uint8 erro
float32 value

